rpaas/plugin.py

```diff
@@ -200,6 +200,76 @@ def block(args):
         sys.exit(1)
 
 
+def nginx_lua_block(value):
+    if value in ('server', 'worker'):
+        return value
+    raise argparse.ArgumentTypeError('Type must be "server" or "worker"')
+
+
+def get_lua_args(args):
+    parser = argparse.ArgumentParser("lua")
+    parser.add_argument("action", choices=["add", "list", "remove"],
+                        help="Action, add or remove blocks from nginx config")
+    parser.add_argument("-s", "--service", required=True, help="Service name")
+    parser.add_argument("-i", "--instance", required=True, help="Instance name")
+    parser.add_argument("-t", "--type", required=False,
+                        help="Lua module type. Should be server or worker.", type=nginx_lua_block)
+    parser.add_argument("-n", "--module-name", required=False, help="Lua module name.")
+    parser.add_argument("-c", "--content", required=False,
+                        help="Raw lua module content")
+    parsed = parser.parse_args(args)
+    if parsed.action == 'add' and (not parsed.module_name):
+        sys.stderr.write("block_name and content are required\n")
+        sys.exit(2)
+    return parsed
+
+
+def lua(args):
```

Comment:
You should add lua entry on dict at available_commands function also to make it works via command line argument